website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -181,6 +181,55 @@ Pass the argument `@isLoading={{true}}` to the item. This will show a “loading
 </Hds::Dropdown>
 ```
 
+### ListItem::Checkmark
+
+For switching context (e.g., organization switchers, project switchers, etc.) use `ListItem::Checkmark`.
+
+```handlebars
+<Hds::Dropdown @listPosition="bottom-left" as |dd|>
+  <dd.ToggleButton @text="HCP Design Sandbox" @color="secondary" />
+  <dd.Checkmark>ACME Org</dd.Checkmark>
+  <dd.Checkmark @selected={{true}}>HCP Design Sandbox</dd.Checkmark>
+  <dd.Footer @hasDivider={{true}}>
+    <Hds::Link::Standalone @icon="list" @text="All Organizations" @color="secondary" @href="#" />
+  </dd.Footer>
+</Hds::Dropdown>
+```
+
+### ListItem::Checkbox
+
+For multi-selection within a form or larger filter pattern use `ListItem::Checkbox`.
+
+```handlebars
+<Hds::Dropdown as |dd|>
```

Comment:
```suggestion
<Hds::Dropdown @listPosition="bottom-left" as |dd|>
```

Reply:
I set them to `bottom-left` initially and then had second thoughts and removed them 😄